src/env.ts

```diff
@@ -0,0 +1,8 @@
+export type Env = {
+  GOOGLE_WEB_RISK_API_KEY: string;
```

Comment:
Is there a way to make these dynamic based on a header?

Like `x-gateway-server-api-key-name` = "GOOGLE_WEB_RISK" => GOOGLE_WEB_RISK_API_KEY environment variable?

Reply:
yeah, I can make it dynamic